puppet/zulip/manifests/voyager.pp

```diff
@@ -27,7 +27,10 @@
   include zulip::memcached
   include zulip::rabbit
   include zulip::redis
-  include zulip::localhost_camo
+  if $::osfamily == debian {
+    # Do not setup camo on systems other than Debian
```

Comment:
"camo is only required on Debian-based systems as part of our migration towards not including camo at all."

Reply:
Also, make the same change in `dockervoyager.pp`.

Reply:
I didn't set it for `dockervoyager.pp` because I'd assume anyone who uses Docker would prefer to use Debian. Though it is a 3-line change so adding it anyway for completeness.

Reply:
Found that camo is not on `dockervoyager.pp`, nor anywhere at docker-zulip.

Reply:
I see.  Well, I guess that's an independent issue, then...